lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1979,3 +1979,56 @@ input[name="subject"] {
   left: 2em;
   right: auto;
 }
+
+.course-specific-container {
+
+  .upload {
+    position: relative;
+
+    button{
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: 1;
+    }
+
+    .fileupload {
+      position: relative;
+      z-index: 2;
+      opacity: 0;
+      height: 30px;
+      width: 105px;
+    }
+  }
+
+  .bordered-fieldset {
+    border: solid 4px #000000;
+    padding: 20px;
+    margin-top: 30px;
+  }
+
+  .error {
+    color: #E40004;
+  }
+
+  .success {
+    color: #008000;
+  }
+
+  .new-section {
+    margin-top: 20px;
+  }
+
+  .select-tag {
+    width: 40%;
+    height: 35px;
+    font-size: 1.1em;
+  }
+
+  .output-box {
+    .loading {
+      font-size: 2em;
+      margin-bottom: 20px;
+    }
+  }
+}
```

Comment:
This SCSS looks good to me. If/When we merge this upstream the px values will have to be converted to use the `$baseline` variable, but should be ok for us right now...